plugins/modules/ec2_instance.py

```diff
@@ -1010,6 +1014,8 @@ def build_volume_spec(params):
     volumes = params.get('volumes') or []
     for volume in volumes:
         if 'ebs' in volume:
+            if 'encrypted' in volume['ebs']:
+                volume['ebs']['encrypted'] = volume['ebs']['encrypted']
```

Comment:
Not sure if we need this.

Reply:
This is needed as the volume has to be encrypted to set the stop-hibernate option.

Reply:
It I am not mistaken, this assignment does nothing. Why should we need this? I tried the PR commenting on these lines and it seems working as expected to me.

Reply:
That was not needed. Removed the lines.